src/pixelator/pixeldataset/precomputed_layouts.py

```diff
@@ -232,15 +232,20 @@ def data():
         for sample_name, layout in precomputed_layouts:
             if layout is None:
                 continue
+            if layout.is_empty:
+                continue
             layout_with_name = layout.lazy.with_columns(
                 sample=pl.lit(sample_name)
             ).pipe(zero_fill_missing_markers, all_markers=all_markers)
             yield layout_with_name
 
-    return PreComputedLayouts(
-        pl.concat(data(), rechunk=False),
-        partitioning=["sample"] + PreComputedLayouts.DEFAULT_PARTITIONING,
-    )
+    try:
+        return PreComputedLayouts(
+            pl.concat(data(), rechunk=False),
+            partitioning=["sample"] + PreComputedLayouts.DEFAULT_PARTITIONING,
```

Comment:
Is DEFAULT_PARTITIONING going to be only used in PreComputedLayouts? If so, I think this is ok. Otherwise, as a default, should we make this variable private and use a getter?

Reply:
My idea was to add it as a constant class variable that can also be used in other parts of the code as needed, and since getters and setters normally deal with instance variables I don't think it's appropriate in this case.